tests/python/gaia-ui-tests/gaiatest/apps/search/regions/browser.py

```diff
@@ -8,7 +8,7 @@
 
 class Browser(Base):
 
-    _browser_app_locator = (By.CSS_SELECTOR, 'div.browser[transition-state="opened"]')
+    _browser_app_locator = (By.CSS_SELECTOR, 'div[data-manifest-name="Browser"][transition-state="opened"]')
```

Comment:
Yeah, I was thinking the same thing, I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1205262 . I'll needinfo you on this.
We also need to have a way of way of being able to pick the 3rd browser instance when there are 4 browser windows open, for example.